Projects/App/Sources/UgglanStore.swift

```diff
@@ -67,6 +69,28 @@ public final class UgglanStore: StateStore<UgglanState, UgglanAction> {
             return performTokenExchange(with: exchangeToken)
         case let .exchangePaymentToken(token):
             return performTokenExchange(with: token)
+        case .validateAuthToken:
+            return client.fetch(query: GraphQL.MemberIdQuery())
+                .valueThenEndSignal
+                .atError(on: .main) { error in
+                    print(error.localizedDescription)
```

Comment:
remove this, but we should probably log here so we can see in Datadog if this happens too often.